src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -5,22 +5,26 @@
 
 <template>
 	<div class="avatar-wrapper" :class="avatarClass" :style="avatarStyle">
-		<div v-if="iconClass" class="avatar icon" :class="[iconClass]" />
-		<div v-else-if="isGuestUser" class="avatar guest">
-			{{ firstLetterOfGuestName }}
-		</div>
-		<div v-else-if="isBot" class="avatar bot">
-			>_
-		</div>
-		<img
-			v-else-if="isFederatedUser && token"
-			:key="avatarUrl"
-			:src="avatarUrl"
-			:width="size"
-			:height="size"
-			:alt="name"
-			class="avatar icon"
-			@error="failed = true">
+		<NcAvatar
+			v-if="isSpecialAvatar"
+			:key="name + '_' + id"
+			class="avatar"
+			:user="name + '_' + id"
+			:url="!isFederatedUser ? undefined : avatarUrl"
+			:icon-class="iconClass"
+			:display-name="name"
+			:disable-tooltip="disableTooltip"
+			disable-menu
+			:hide-status="!showUserStatus"
+			:verbose-status="!showUserStatusCompact"
+			:preloaded-user-status="preloadedUserStatus ?? emptyUserStatus"
```

Comment:
Should be enough, we don't need a computed for it
```suggestion
			:preloaded-user-status="preloadedUserStatus ?? {}"
```

Reply:
oh I see you are adding an empty object 🙈